howto/anbox/develop-addon.md

```diff
@@ -0,0 +1,45 @@
+Developing and testing addons using AMS may be time-consuming. Instead, a container with `--devmode` enabled can provide a safe environment to test addons and their scripts without having to upload the addon to the AMS.
```

Comment:
Is it `AMS` or `the AMS`?
Because we first have `using AMS may be time-consuming` and then `upload the addon to the AMS`.